src/components/Extension.tsx

```diff
@@ -24,41 +24,34 @@
 
  */
 
-import React, { useState } from "react"
-import { ComponentsProvider, MessageBar } from "@looker/components"
+import React from "react"
+import { ComponentsProvider } from "@looker/components"
 import { DiagramFrame } from "./DiagramFrame/DiagramFrame"
 import { useSelectExplore, usePathNames } from "../utils/routes"
-import { DiagramError } from "../utils/fetchers"
 import { VIEW_OPTIONS_HIDDEN_DEFAULT, VIEW_OPTIONS_FIELDS_DEFAULT,
 } from '../utils/constants'
 
 export const Extension: React.FC = () => {
-  const [diagramError, setDiagramError] = useState<DiagramError | undefined>(undefined)
-  const [selectedBranch, setSelectedBranch] = React.useState("")
   const { modelName, exploreName, fullPage } = usePathNames()
   const [hiddenToggle, setHiddenToggle] = React.useState(VIEW_OPTIONS_HIDDEN_DEFAULT)
   const [displayFieldType, setDisplayFieldType] = React.useState(VIEW_OPTIONS_FIELDS_DEFAULT)
-  const { unfilteredModels, modelDetail, dimensions } = useSelectExplore(hiddenToggle, displayFieldType, selectedBranch, diagramError, setDiagramError)
+  const { unfilteredModels, modelDetail, dimensions } = useSelectExplore(hiddenToggle, displayFieldType)
   return (
   <ComponentsProvider themeCustomizations={{
+    fontFamilies: { brand: 'Google Sans' },
```

Comment:
"Google Sans" is design spec. No embedded or spartan use cases (yet!).

I suspect I won't be able to use your change until 21.8 is ESR

Reply:
You can, you will get an undefined value for the hostType which means you should use roboto.